fileupload/client_test.go

```diff
@@ -55,3 +55,40 @@ func TestFileUploadNewThenGet(t *testing.T) {
 		t.Errorf("(GET) File upload id %q does not match expected id %q\n", target.ID, res.ID)
 	}
 }
+
+func TestFileUploadList(t *testing.T) {
+	f, err := os.Open("test_data.pdf")
```

Comment:
Yep, it was added for tests for file uploads when they were originally added to the Go bindings (here: https://github.com/stripe/stripe-go/blob/master/fileupload/test_data.pdf)